### DISCUSSION TITLE
Time to merge a PR

### DISCUSSION THREAD

**Discussion by RohitRaj279** [sanjay-kv/Open-source-Practice#914] [Q&A] [answered]
What is the approximated time required for a PR to be merged?

**Comment by MrCodYrohit** [comment, marked as answer]
It depends on Project admin and can vary from 1 day to 3-4 days depending on the length of your contribution and timings of PR raised(weekends).

**Comment by RohitRaj279** [comment]
can u give more insights if possible?

**Comment by lovelymahor** [comment]
The approximate time required for a pull request (PR) to be merged varies depending on several factors, including the project's size, complexity, maintainers' availability, and the quality of the contribution. Generally, smaller and less complex changes may be merged more quickly, sometimes within a few days. However, larger changes or those requiring extensive review may take longer, ranging from a few days to several weeks. It's essential to be patient and proactive in addressing feedback to facilitate the merging process.
